draft-ietf-quic-manageability.md

```diff
@@ -91,13 +91,13 @@ is exposed by QUIC to the network, requirements and assumptions that the QUIC
 design with respect to network treatment, and a description of how common
 network management practices will be impacted by QUIC.
 
-Since QUIC's wire image {{?WIRE-IMAGE=RFC8546}} is integrity protected and not
-modifiable on path, in-network operations are not possible without terminating
-the QUIC connection, for instance using a back-to-back proxy. Proxy operations
-are not in scope for this document. A proxy can either explicit identify itself
-as providing a proxy service, or may share the TLS credentials to authenticate
-as the server and (in some cases) client acting as a front-facing instance for
-the endpoint itself.
+Since QUIC's wire image {{?WIRE-IMAGE=RFC8546}} is integrity protected,
+in-network operations that depend on modification of data are not possible
+without the cooperation of an endpoint. Network operation practices that alter
+data are only possible if performed as a QUIC endpoint, which might be possible
+with the introduction of a proxy, which either autenticates itself as a
```

Comment:
```suggestion
with the introduction of a proxy, which either authenticates as an endpoint.
```

It's not necessary to distinguish acting **for** an endpoint and acting **as** an endpoint like this.  As far as the peer is concerned, they are the same thing.

Reply:
```suggestion
with the introduction of a proxy which authenticates as an endpoint.
```